fastavro/_write.pyx

```diff
@@ -121,7 +123,9 @@ cdef inline write_bytes(bytearray fo, bytes datum):
 cdef inline write_utf8(bytearray fo, datum):
     """A string is encoded as a long followed by that many bytes of UTF-8
     encoded character data."""
-    write_bytes(fo, utob(datum))
+    b_datum = utob(datum)
```

Comment:
I see that this would save a function call, but I would assume that the function call wouldn't be as expensive in cython as it is in python. Is there a measurable difference here?

Reply:
Here to make up for the performance loss from the changed signature of write_bytes.  I'm guessing as a result of casting away from the known bytes-type here.